### ISSUE TITLE
Building manually sets a wrong save path

### ISSUE THREAD

**Issue by cooperdk** [naelstrof/KoboldKare#213]
In the stable, the save goes to AppData\LocalLow\Naelstrof\KoboldKare\saves
Building from github, the save goes to AppData\LocalLow\Naelstrof\KoboldKare\defaultUser\saves

The mods folder stays the same.

Not that it matters, because it's not possible to load a game from neither the stable nor the test build (WTF??)

**Comment by naelstrof** [comment]
Ah yeah sorry, I'm moving things around for Steam Cloud build support. There's a few bugs left.

**Comment by naelstrof** [comment]
> Not that it matters, because it's not possible to load a game from neither the stable nor the test build (WTF??)

Oh I misread this, it should be possible to load the saves from the test build just fine. There's one edge-case: if you launch the game without steam, it'll move all your saves to the "defaultUser" folder. Then if you launch with steam, it'll give you a new user folder which will be empty.

**Comment by cooperdk** [comment]
Sorry, I forgot to remove that line. It is indeed possible.
My original issue was that starting the test build for the first time did a number of stuff to the mods. It moved the mods to other folders in the mod directory, while i think it should have moved them to the addressables directory. It caused both the stable and the test build to malfunction. Once I removed those two directories and ran again, it worked fine (apart from the save directory, where I just copied the savegames over, I didn't want to make a junction cause I wanted to test it further.
Now it's explained, so thanks :-)
